diff -rc ExternalEditor/zopeedit.py ExternalEditor.patch/zopeedit.py
*** ExternalEditor/zopeedit.py	2005-03-04 14:08:41.676074120 +0100
--- ExternalEditor.patch/zopeedit.py	2005-03-04 14:08:55.607956152 +0100
***************
*** 98,103 ****
--- 98,107 ----
                      config_path = global_config
              else:
                  config_path = os.path.expanduser('~/.zope-external-edit')
+                 global_config = '/etc/ZopeEdit.ini'
+                 if not os.path.exists(config_path) \
+                    and os.path.exists(global_config):
+                     os.system("cp %s %s" % (global_config, config_path))
                  
              self.config = Configuration(config_path)
  
